docs/source/guide/tutorials/base/demo.rst

```diff
@@ -1,47 +1,65 @@
-How to run the demo with exportable code
-========================================
+How to run the demo
+===================
 
-In this tutorial we will show how run :doc:`trained <how_to_train/index>` and :doc:`deployed <deploy>` model in demostrarion mode. It allows us to apply our model on the custom data or the online footage from a web camera and see how it will work in the real-life scenario.
+In this tutorial we will show how run :doc:`trained <how_to_train/index>` model inside OTX repository in demonstration mode.
+It allows us to apply our model on the custom data or the online footage from a web camera and see how it will work in the real-life scenario.
```

Comment:
Please, add a disclaimer that ~ "This tutorial is for object detection model, however for other tasks the functionality remains the same - you only need to change input data on your own"

Reply:
Also put a link describing WGISD dataset and describe it, please, like :"we will be using `WGISD <link>`_ dataset from `:doc: object detection tutorial <link>`